docs: js, vuejs and react js links (#141)
sdtrdev/source-pocket: docs/web-development/vuejs.md

## Beginner

- [🎥 Vue JS Crash Course 2021 | Traversy Media 🎥](https://youtu.be/qZXt1Aom3Cs "Vue JS Crash Course 2021")

> - [Getting Started with Vue - An Overview and Walkthrough Tutorial – Tania Ras](https://www.taniarascia.com/getting-started-with-vue)

Reviewer: ```suggestion
- [Getting Started with Vue - An Overview and Walkthrough Tutorial – Tania Rascia](https://www.taniarascia.com/getting-started-with-vue)
```

## Intermediate

- [Vue.js Guide](https://vuejs.org/v2/guide/ "Vue.js Guide")
- [Utilize the File Structure to Decide When to Use Vue.js Slots - Markus Oberlehner](https://markus.oberlehner.net/blog/utilize-the-file-structure-to-decide-when-to-use-vuejs-slots/)
- [Async Vue.js Components - Vue.js Tutorials](https://vueschool.io/articles/vuejs-tutorials/async-vuejs-components)

> - [How to Server Side Rendering (SSR) with Vue.js? | by Taha Selim Aksakal | T](https://medium.com/swlh/how-to-server-side-rendering-ssr-with-vue-js-71cedce7c299)

Reviewer: ```suggestion
- [How to Server Side Rendering (SSR) with Vue.js? | by Taha Selim Aksakal | The Startup](https://medium.com/swlh/how-to-server-side-rendering-ssr-with-vue-js-71cedce7c299)
```

- [Drag and Drop File Upload in Vue.js](https://www.raymondcamden.com/2019/08/08/drag-and-drop-file-upload-in-vuejs)

## Advanced

- [Implementing the Builder Pattern in Vue.js Part 2: Forms - Markus Oberlehner](https://markus.oberlehner.net/blog/implementing-the-builder-pattern-in-vue-forms/)

> - [Managing Complex Waiting Experiences on Web UIs | by Fatih Kadir Akın | Med](https://medium.com/@fkadev/managing-complex-waiting-experiences-on-web-uis-29534d2d92a8)

Reviewer: ```suggestion
- [Managing Complex Waiting Experiences on Web UIs | by Fatih Kadir Akın | Medium](https://medium.com/@fkadev/managing-complex-waiting-experiences-on-web-uis-29534d2d92a8)
```
